### DISCUSSION TITLE
Create a Cross Platform Mobile App With Flutter in Under 5 Minutes! – Benjamin Carlson

### DISCUSSION THREAD

**Discussion by giscus** [bjcarlson42/benjamincarlson.io#52] [General]
# Create a Cross Platform Mobile App With Flutter in Under 5 Minutes! – Benjamin Carlson

Learn how to create a cross platform android and IOS mobile app using dart and Flutter.

https://www.benjamincarlson.io/blog/create-a-cross-platform-app-with-flutter

**Comment by VijayKodibailu** [comment]
awesome!
